src/_pytest/timing.py

```diff
@@ -20,6 +20,38 @@
     from pytest import MonkeyPatch
 
 
+class Instant:
+    """
+    Provides a measurement of timing between different points in the code.
+
+    Useful to compute the elapsed time between two points in time,
+    using a performance counter, and the beginning/end in actual times (as seconds since epoch).
+
+    Inspired by Rust's `std::time::Instant`.
+    """
+
+    def __init__(self) -> None:
+        import _pytest.timing  # noqa: PLW0406
```

Comment:
This one should  not use import at all

Reply:
Can you elaborate why? To me it should, as we will use this in production, so being able to mock it in some tests seems desirable and what one would expect.

Reply:
the mock monkeypatches the modules globals to begin with, so there is no reason to self-import - the globals will replace

Reply:
Ahh right right, thanks, you are right.